README.md

```diff
@@ -70,7 +70,7 @@ class Page extends Resource
 
 The package supports four kinds of dependencies:
 
-1. `->dependsOn('field', 'value')`
+1. `->dependsOn('field', 'value')` or `->dependsOn('field', ['value1', 'value2'])` (using 'or' matching)
```

Comment:
It's always `or` matching, there is no `and`.

Reply:
@wize-wiz If I make this one change you will merge?